assets/model_monitoring/components/tests/e2e/resources/mltable_iris_baseline_int_data_type/iris_baseline.csv

```diff
@@ -11,4 +11,4 @@ sepal_length,sepal_width,petal_length,petal_width,target
 4,3,1,5,virginica
 5,3,1,6,versicolor
 4,3,1,5,virginica
-5,3,1,6,versicolor
\ No newline at end of file
+5,3,1.0,6,versicolor
```

Comment:
This is already the single value test file? Why adding a new one and changing this one?

Reply:
This is technically not the single-value test file because the corresponding production data (DATA_ASSET_IRIS_PREPROCESSED_MODEL_INPUTS_NO_DRIFT_INT_DATA) for this reference file doesn't have a single-value column and so the test doesn't check for the single distinct value histogram being built correctly. Also, I changed it because currently that column is being inferred as boolean and I don't think that was the intention of the testcase but I can change it back if we want.